index.js

```diff
@@ -0,0 +1,13 @@
+import BookCollection from './modules/book.js';
+import { DateTime } from './modules/luxon.js';
+
+const bookCollection = new BookCollection();
+document
+  .getElementById('add-button')
+  .addEventListener('click', bookCollection.addBook);
+bookCollection.displayBooks();
+
+const now = DateTime.now();
+const date = now.toLocaleString(DateTime.DATETIME_FULL);
+
+document.getElementById('date').textContent = date;
```

Comment:
- [ ] [OPTIONAL] Your code is really impressive and the design is requisite! 💯 🥇 but you can also improve user experience by making your time to update in real-time just like in normal setting. without having to refresh. For this, I think [ `setInterval()`](https://www.w3schools.com/jsref/met_win_setinterval.asp)method will be a great tool. :; 💪